hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/PrisonerDetailResource.kt

```diff
@@ -30,8 +30,7 @@ class PrisonerDetailResource(private val prisonerDetailService: PrisonerDetailSe
 
   @Operation(
     summary = "Find prisoners by exact or wildcard terms for specified fields and return a paginated result set",
-    description = """
-      Search terms and identifiers can be provided in either or mixed case and are converted to the appropriate case.
+    description = """Search terms and identifiers can be provided in either or mixed case and are converted to the appropriate case.
```

Comment:
running onto two lines means that the whole description gets treated as `code` and in purple